README.md

```diff
@@ -54,7 +54,14 @@ This build starts by pulling preprocessed sequence and metadata files from:
 * https://data.nextstrain.org/files/dengue/sequences_denv4.fasta.zst
 * https://data.nextstrain.org/files/dengue/metadata_denv4.tsv.zst
 
-The above datasets have been preprocessed and cleaned from GenBank and are updated at regular intervals. 
+The above datasets have been preprocessed and cleaned from GenBank and are updated at regular intervals from the ingest folder.
```

Comment:
This section is a little unclear to me. It seems to imply that one should run the code here (`nextstrain build ingest`) to pull the preprocessed sequence and metadata files. I'd instead replace the code block with a sentence along the lines of "See `./ingest/README.md` for how these files are generated."